config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -771,39 +779,45 @@ private TrackingEventProcessorConfiguration trackingEventProcessorConfig(String
         return tepConfigs.getOrDefault(name, defaultTrackingEventProcessorConfiguration).get();
     }
 
-    private SubscribingEventProcessor subscribingEventProcessor(String name,
-                                                                EventHandlerInvoker eventHandlerInvoker,
-                                                                SubscribableMessageSource<? extends EventMessage<?>> messageSource) {
+    /**
```

Comment:
Missing some JavaDoc here too! I might help you out here though.

Reply:
what would you recommend to write here?